firestore/src/ios/firebase_metadata_provider_desktop.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_IOS_FIREBASE_METADATA_PROVIDER_DESKTOP_H_
+#define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_IOS_FIREBASE_METADATA_PROVIDER_DESKTOP_H_
```

Comment:
It seems the file name doesn't consist with the path. desktop files under ios path?